.github/workflows/CI.yml

```diff
@@ -903,6 +903,63 @@ jobs:
         run: |
           cd testing-framework/terraform/eks && terraform destroy -auto-approve
 
+  e2etest-eks-arm64:
+    runs-on: ubuntu-latest
+    needs: [get-testing-suites, e2etest-release, e2etest-preparation]
```

Comment:
We might not need the e2etest-release since we don't have any depend output on that job.

Reply:
That is where the we push to ecr

Reply:
Forgot the name. I thought the ```e2etest-preparation``` comes before ```e2e-release```